src/lib/userPerformanceMetrics.ts

```diff
@@ -159,3 +161,8 @@ export function getDomComplete() {
   if (!timingAvailable) return null
   return sanitizedMetrics(perf.timing.requestStart, perf.timing.domComplete)
 }
+
+export async function getTTI() {
+  if (!timingAvailable) return null
```

Comment:
This check probably needs to be more restrictive, should check the corresponding API, or maybe browser versions.

Reply:
I don't think it really needs to be any more restrictive. The polyfill itself does all the feature checking that's necessary.

Reply:
Indeed: https://github.com/GoogleChromeLabs/tti-polyfill/blob/4b5058efb695f2f51aa6f586d9a46dc9c0207f30/src/index.js#L35